python/TestHarness/testers/Tester.py

```diff
@@ -61,21 +62,28 @@ def __init__(self, name, params):
         MooseObject.__init__(self, name, params)
         self.specs = params
 
+        # Bool if test can run
+        self._runnable = None
+
+        ### several variables needed when performing processResults
+        self.__start_time = clock()
+        self.__end_time = None
+        self.__exit_code = 0
+        self.__std_out = ''
```

Comment:
This is good! You are still setting these from outside the Tester though. I commented on that earlier.